### DISCUSSION TITLE
Partial Parallax?

### DISCUSSION THREAD

**Discussion by tyteen4a03** [pmndrs/react-spring#1680] [Support]
Is it possible for a Parallax effect to only apply within a specific section of the page (i.e. HTML elements before and after the `<Parallax>` scroll normally), or do I have to put the entire page into Parallax layers?

(It could be a problem with my markup as currently when I scroll through `<Parallax>` the HTML that's supposed to come after the Parallax is showing. When I scroll back up, the Parallax breaks.)

**Comment by kindoflew** [comment]
(I answered this in the Discord server, but i'll say it here too for more visibility)

`Parallax` is a scrollable, absolutely positioned container. it will have it's own scrollbar, separate from the browser's. So, the short answer is: yes, everything would need to be inside it.

anything that you don't want parallaxing should be inside a `ParallaxLayer` with its speed set to `0` and the `offset` would be wherever you want the content placed on the y-axis.
